.github/workflows/test-chart.yml

```diff
@@ -0,0 +1,102 @@
+name: Test Chart
+
+on:
+  push:
+  pull_request:
+
+env:
+  # The actual version will be injected to the chart by the task that builds the chart
+  DUMMY_VERSION: 1.2.3
+
+jobs:
+  test-e2e:
+    name: Run on Ubuntu
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone the code
+        uses: actions/checkout@v4
+
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+
+      - name: Install the latest version of kind
+        run: |
+          curl -Lo ./kind https://kind.sigs.k8s.io/dl/latest/kind-linux-amd64
+          chmod +x ./kind
+          sudo mv ./kind /usr/local/bin/kind
+
+      - name: Verify kind installation
+        run: kind version
+
+      - name: Create kind cluster
+        run: kind create cluster
+
+      - name: Prepare integration-service
+        run: |
+          go mod tidy
+          export IMG=integration-service:v$DUMMY_VERSION
+          yq -i '.controllerManager.container.image = env(IMG)' dist/chart/values.yaml
+          make img-build IMG=$IMG
+          kind load docker-image $IMG
+
+      - name: Verify Helm installation
+        run: helm version
+
+      - name: Lint Helm Chart
+        run: |
+          helm lint ./dist/chart
+
+      - name: Install cert-manager via Helm
+        run: |
+          helm repo add jetstack https://charts.jetstack.io
+          helm repo update
+          helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --set installCRDs=true
+
+      - name: Wait for cert-manager to be ready
+        run: |
+          kubectl wait --namespace cert-manager --for=condition=available --timeout=300s deployment/cert-manager
+          kubectl wait --namespace cert-manager --for=condition=available --timeout=300s deployment/cert-manager-cainjector
+          kubectl wait --namespace cert-manager --for=condition=available --timeout=300s deployment/cert-manager-webhook
+
+      - name: Install Prometheus Operator CRDs
+        run: |
+          helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+          helm repo update
+          helm install prometheus-crds prometheus-community/prometheus-operator-crds
+
+      - name: Install Prometheus via Helm
+        run: |
+          helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
```

Comment:
Why are we duplicating code from line 65 and 66?

Reply:
Thanks for pointing this out. I suspect that those two sections were auto-generated pre-commented-out, allowing you to uncomment either independently, so the lines are repeated to allow them to work either way.

As we use both, there's indeed no need to repeat those lines. I'll have it addressed.